hr_attendance_calendar/models/hr_attendance_day.py

```diff
@@ -96,6 +100,27 @@ class HrAttendanceDay(models.Model):
     ##########################################################################
     #                             FIELDS METHODS                             #
     ##########################################################################
+    @api.multi
+    @api.depends('attendance_ids')
+    def _compute_has_logged_hours(self):
+        for att_day in self:
+            # > 1 because when we calculate this, we're creating an attendance
```

Comment:
Are you sure this is still the case? Normally the compute method is called when the field is accessed for the first time (when the field is not stored).

Reply:
I feel this field is useless, when needed you can simply do `if att_day.attendance_ids` to check if there are logged hours

Reply:
The field is not used in hr.attendance.day model, but a related field of this is needed in hr.attendance, to show the wizard only if there are already logged hours.
And for len > 1, the field depends on attendance_ids, so it's recomputed each time an attendance_day has a new attendance